Composer/packages/client/locales/en-US.json

```diff
@@ -0,0 +1,1259 @@
+{
+  "0_bytes_a1e1cdb3": {
```

Comment:
These IDs are generated when you call formatMessage like this formatMessage('hello'), we could have formatMessage({id: 'mycomponent.helloMessage, default: 'hello'}).  Do you think semantic IDs help give context to localization?